MNE/disp3D/3DObjects/brain/brainsurfacesettreeitem.cpp

```diff
@@ -243,6 +250,70 @@ bool BrainSurfaceSetTreeItem::addData(const MNESourceSpace& tSourceSpace, Qt3DCo
 
 //*************************************************************************************************************
 
+BrainRTSourceLocDataTreeItem* BrainSurfaceSetTreeItem::addData(const MNESourceEstimate& tSourceEstimate, const MNEForwardSolution& tForwardSolution)
+{
+    if(!tSourceEstimate.isEmpty()) {
+        //Add source estimation data as child
+        if(this->findChildren(Data3DTreeModelItemTypes::RTSourceLocDataItem).size() == 0) {
+            //If rt data item does not exists yet, create it here!
+            if(!tForwardSolution.isEmpty()) {
+                m_pBrainRTSourceLocDataTreeItem = new BrainRTSourceLocDataTreeItem();
+
+                QList<QStandardItem*> list;
+                list << m_pBrainRTSourceLocDataTreeItem;
+                list << new QStandardItem(m_pBrainRTSourceLocDataTreeItem->toolTip());
+                this->appendRow(list);
+
+                connect(m_pBrainRTSourceLocDataTreeItem, &BrainRTSourceLocDataTreeItem::rtVertColorChanged,
+                        this, &BrainSurfaceSetTreeItem::onRtVertColorChanged);
+
+                //Divide into left right hemi
+                QList<QStandardItem*> itemList = this->findChildren(Data3DTreeModelItemTypes::HemisphereItem);
+
+                BrainSurfaceTreeItem* pSurfaceTreeItemLeft = new BrainSurfaceTreeItem();
```

Comment:
Don't initialize with new here --> in line 281 the pointers are assigned again and the with "new" allocated memory is leaking!!!